src/preferences/dialog/dlgprefdeckdlg.ui

```diff
@@ -14,6 +14,7 @@
    <string>Deck Preferences</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
+
```

Comment:
I am unsure but does this file contain unrelated changes?

Reply:
Nope. To make that traceabe I did the reorder commit where the line count didn't change (it did, I added 22 blank lines for readability) and one where I added the new widgets.
Reorder https://github.com/mixxxdj/mixxx/pull/4816/commits/82cede4d7593135ae7721ca5d125c7f849a4247b
```
git diff --stat 283139b08c 82cede4d75
 src/preferences/dialog/dlgprefdeckdlg.ui | 646 +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++-------------------------------------------------------------------------------------------
 1 file changed, 334 insertions(+), 312 deletions(-) <--- + 22 blank lines
```

I will add fixup commits to make the necessary changes.